FreeRTOS/Demo/CORTEX_MPS2_QEMU_IAR/FreeRTOSConfig.h

```diff
@@ -107,6 +107,10 @@ See http://www.FreeRTOS.org/RTOS-Cortex-M3-M4.html. */
 version. */
 #define configUSE_PORT_OPTIMISED_TASK_SELECTION 1
 
+/* The Win32 target is capable of running all the tests tasks at the same
+ * time. */
+#define configRUN_ADDITIONAL_TESTS				1
```

Comment:
Does `configRUN_ADDITIONAL_TESTS` only apply to message buffer tests? If so, the name seems too generic.

Reply:
At the moment it only applies to message buffers - but the intent is for it to apply whenever additional tests are added into an existing set of tests.  Large devices (Zynq, Win32, POSIX/Linux, QEMU) can run all the tests so set configRUN_ADDITIONAL_TESTS to 1.  Smaller devices that already run the message buffer tests may not have the resources to create the two additional tasks added so keeping configRUN_ADDITIONAL_TESTS defaulting to 0 ensures they still run their suites of tests too.

Reply:
What I did here is - created a flag to control each test suite/group: https://github.com/aggarg/FreeRTOS/blob/template/FreeRTOS/Demo/ThirdParty/Template/TestRunner.c#L114

Large devices can enable all groups while smaller devices can enable one or more at a time.